### PR TITLE
Rename KNITRO to Artelys knitro

### PR DESCRIPTION
**Pull request by yeesian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,16 +3,16 @@ KNITRO.jl
 
 The KNITRO.jl package provides an interface for using the [Artelys Knitro solver](http://artelys.com/en/optimization-tools/knitro) from [Julia](http://julialang.org/). You cannot use KNITRO.jl without having purchased and installed a copy of Knitro from [Artelys](http://artelys.com/). This package is available free of charge and in no way replaces or alters any functionality of Artelys Knitro solver.
 
-Knitro functionality is extensive, so coverage is incomplete, but the basic functionality for solving linear, nonlinear, and mixed-integer programs is provided. Documentation is available on [ReadTheDocs](http://knitrojl.readthedocs.org/en/latest/knitro.html).
+Artelys Knitro functionality is extensive, so coverage is incomplete, but most functionality for solving linear, nonlinear, and mixed-integer programs is provided. Documentation is available on [ReadTheDocs](http://knitrojl.readthedocs.org/en/latest/knitro.html).
 
-*The Knitro wrapper for Julia is community driven and not officially supported by Artelys. If you are an Artelys customer interested in official support for Julia, let them know!*
```
Comment (mlubin) [inline README.md]: "Artelys customer" is correct, not "Artelys Knitro customer"

File: README.md [hunk 2]
```diff
@@ -40,11 +40,11 @@ Setting up Knitro on Windows
 
 Note that currently *only 64-bit* Windows is supported. That is, you must use 64-bit Julia and install the Win64 version of Knitro.
 
-1. First, you must obtain a copy of the Knitro software and a license; trial versions and academic licenses are available.
+1. First, you must obtain a copy of the Artelys Knitro solver and a license; trial versions and academic licenses are available.
 
-2. Once Knitro is installed on your machine, add the directory containing ``knitro.dll`` to the `PATH` environment variable, as described in the Knitro documentation.
+2. Once Artelys Knitro is installed on your machine, add the directory containing ``knitro.dll`` to the `PATH` environment variable, as described in the Artelys Knitro documentation.
 
-3. To activate Knitro for your computer you will need a valid license file (which looks like `ziena_lic_*.txt`). The simplest procedure is to copy each license into your `HOME` directory.
```
Comment (mlubin) [inline README.md]: Has the name of the license file changed?

File: doc/conf.py [hunk 3]
```diff
@@ -14,13 +14,6 @@
 
 import sys, os
 
-on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
```
Comment (mlubin) [inline doc/conf.py]: Why remove this?

Comment (yeesian) [inline doc/conf.py]: It stopped working for me. E.g. https://github.com/codesy/codesy/pull/84/files

File: README.md (hunk 2 above)
Comment (yeesian) [inline README.md]: Just to future-proof the README

File: doc/conf.py (hunk 3 above)
Comment (mlubin) [inline doc/conf.py]: You need to install ``sphinx_rtd_theme`` if you want to build the docs locally.

Comment (mlubin) [inline doc/conf.py]: ``pip install sphinx_rtd_theme``

Comment (yeesian) [inline doc/conf.py]: :ok_hand:

**Comment by mlubin** [comment]
Thanks!

**Pull request merged**